examples/resources/nobl9_alert_policy/resource.tf

```diff
@@ -29,3 +29,22 @@ resource "nobl9_alert_policy" "this" {
   }
 }
 
+resource "nobl9_alert_policy" "this" {
+  name         = "${nobl9_project.this.name}-slow-burn"
+  project      = nobl9_project.this.name
+  display_name = "${nobl9_project.this.display_name} Slow burn (1x12h and 2x15min)"
```

Comment:
I think Peter mentioned that we should include in the terraform documentation both slow burn and fast burn the same way as we do in presets (here we have only slow burn), is that still relevant?